catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/formulaeditor/FormulaEditorRenameVariableTest.java

```diff
@@ -0,0 +1,131 @@
+import org.catrobat.catroid.uiespresso.util.rules.BaseActivityInstrumentationRule;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.Espresso.pressBack;
+import static android.support.test.espresso.action.ViewActions.clearText;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.action.ViewActions.longClick;
+import static android.support.test.espresso.action.ViewActions.typeText;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+
+import static org.catrobat.catroid.uiespresso.content.brick.utils.BrickDataInteractionWrapper.onBrickAtPosition;
+import static org.catrobat.catroid.uiespresso.util.actions.CustomActions.setCheckboxChecked;
+
+@RunWith(AndroidJUnit4.class)
+public class FormulaEditorRenameVariableTest {
+	@Rule
+	public BaseActivityInstrumentationRule<ScriptActivity> baseActivityTestRule = new
+			BaseActivityInstrumentationRule<>(ScriptActivity.class, true, false);
+
+	@Before
+	public void setUp() throws Exception {
+		Script script = BrickTestUtils.createProjectAndGetStartScript("FormulaEditorRenameVariableTest");
+		script.addBrick(new ChangeSizeByNBrick(0));
+		baseActivityTestRule.launchActivity(null);
+	}
+
+	private String variableNameOld = "variableOld";
+	private String variableNameNew = "variableNew";
+
+	@Category({Cat.AppUi.class, Level.Smoke.class})
+	@Test
+	public void renameVariable() {
+		onBrickAtPosition(0).checkShowsText(R.string.brick_when_started);
+		onBrickAtPosition(1).checkShowsText(R.string.brick_change_size_by);
+		onBrickAtPosition(1).onChildView(withId(R.id.brick_change_size_by_edit_text)).perform(click());
+
+		onView(withId(R.id.formula_editor_keyboard_data)).perform(click());
+		onView(withId(R.id.button_add)).perform(click());
+		onView(withId(R.id.dialog_formula_editor_data_name_edit_text)).perform(typeText(variableNameOld));
+		onView(withText(R.string.ok)).perform(click());
+
+		onView(withText(variableNameOld)).perform(click());
+		onView(withId(R.id.formula_editor_edit_field)).check(matches(withText(getUserVariableEditText(variableNameOld))));
+
+		onView(withId(R.id.formula_editor_keyboard_data)).perform(click());
+		onView(withText(variableNameOld)).perform(longClick());
+		onView(withText(R.string.rename)).perform(click());
+		onView(withId(R.id.dialog_formula_rename_variable_name_edit_text))
+				.perform(clearText(), typeText(variableNameNew));
+		onView(withText(R.string.ok)).perform(click());
+		pressBack();
+
+		onView(withId(R.id.formula_editor_edit_field)).check(matches(withText(getUserVariableEditText(variableNameNew))));
```

Comment:
for the sake of readability can you please have the .check or .perform in a new line